l10n_de_tax_statement/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Onestein (<https://www.onestein.eu>)
+# Copyright 2019 BIG-Consulting GmbH(<http://www.openbig.org>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'German VAT Statement',
+    'version': '11.0.1.1.0',
+    'category': 'Localization',
+    'license': 'AGPL-3',
+    'author': 'OpenBIG.org, Onestein, Odoo Community Association (OCA)',
+    'website': 'https://github.com/openbig/l10n-germany',
+    'depends': [
+        'account_invoicing',
+        'account_tax_balance',
+        'l10n_de',
```

Comment:
```suggestion
```

Reply:
I have applied your proposed change. Thanks a lot for your intervention!